stock_picking_backorder_strategy/tests/test_backorder_strategy.py

```diff
@@ -1,10 +1,10 @@
 # Copyright 2018 ACSONE SA/NV
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo.tests import common
+from odoo.tests.common import TransactionCase
```

Comment:
Absolutly non blocking: you've got `TransactionCase` in `odoo.tests` namespace as well.